### ISSUE TITLE
Tests: add token management coverage

### ISSUE THREAD

**Issue by anastasiyaig** [status-im/desktop-qa-automation#609]
Ordering/drag&drop

https://ethstatus.testrail.net/index.php?/suites/view/5308&group_by=cases:section_id&group_order=asc&display_deleted_cases=0&group_id=51546

Assets sorting: non-community

https://ethstatus.testrail.net/index.php?/suites/view/5308&group_by=cases:section_id&group_order=asc&display_deleted_cases=0&group_id=51542

Collectibles: Community minted

https://ethstatus.testrail.net/index.php?/suites/view/5308&group_by=cases:section_id&group_order=asc&display_deleted_cases=0&group_id=51543

Each link has multiple cases but i believe we can compress each section into a single autotest

**Comment by Valentina1133** [comment]
Assets sorting: non-community - done

**Comment by Valentina1133** [comment]
I faced 2 bugs during working on this task. So I changed status to Blocked/On hold until they are fixed

**Comment by anastasiyaig** [comment]
both issues are fixed, please continue working on that

**Comment by Valentina1133** [comment]
https://github.com/status-im/status-desktop/issues/14642 collectibles sorting test is blocked by this bug

**Comment by Valentina1133** [comment]
Assets ordering merged to master. Collectibles sorting is on hold, there is a bug
